etc/leapp/transaction/to_remove

```diff
@@ -12,3 +12,5 @@ python2-virtualenv
 squid
 # python-docs - https://bugzilla.redhat.com/show_bug.cgi?id=1703605
 python-docs
+
```

Comment:
Please add a comment here as of why we remove python-tools, that's helpful when trying to resolve problems

Reply:
The thing is, I don't actually know.  I know the ref should be here, I was expecting @bocekm or @pirat89 to chip in.

I probably should have marked the PR as WIP....

Reply:
@AloisMahdal not sure what is still unclear on this approach...

@vinzenz The same reason as `python2-devel` in the same list.

Reply:
@pirat89 I was responding to Vinny pointing out missing **reference**.  Whether I understand the mechanics of how the transaction is broken is irrelevant.

Frankly, I even think I made that pretty clear even in the OP.  You seem to be missing this constantly. Understanding some technical aspects issue is one thing, referencing a decision transparently (ie. linking to BZ/gh) is another one.  My goal is the second one.